### PR TITLE
Update Changelog.lua to v241

### PR DESCRIPTION
**Pull request by ccuser44**
## Notice
Please make sure you read the Contributing Guidelines before submitting a pull request. https://github.com/Epix-Incorporated/Adonis/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by Dimenpsyonal** [comment]
No

**Comment by PurpleCreativity** [comment]
No

**Comment by ccuser44** [comment]
Yes

**Comment by Dimenpsyonal** [comment]
I very specifically didn't add these

**Comment by ccuser44** [comment]
> I very specifically didn't add these

Done

**Review by watameln — APPROVED** [review]

**Review by watameln — APPROVED** [review]

**Review by watameln — APPROVED** [review]

**Comment by watameln** [comment]
i will approve all changes

**Review by watameln — APPROVED** [review]

**Review by watameln — APPROVED** [review]

**Review by watameln — APPROVED** [review]

**Review by watameln — APPROVED** [review]

**Review by watameln — APPROVED** [review]

**Comment by ccuser44** [comment]
> You meanie smh. Unmergable

Fixed

**Review by watameln — APPROVED** [review]

**Comment by Dimenpsyonal** [comment]
No.

**Review by watameln — APPROVED** [review]

**Comment by ccuser44** [comment]
> No.

Why?

**Comment by moo1210** [comment]
Wow really can I be on it too

**Comment by Dimenpsyonal** [comment]
Yea sure why not

**Comment by Dimenpsyonal** [comment]
I think the @ should be the maintainer that released it instead of the changelog author but ok

File: MainModule/Server/Shared/Changelog.lua [hunk 1]
```diff
@@ -3,18 +3,24 @@ return {
...
-	":BuyItem now only blocks limiteds (#1223)";
-	"Coasterteam various fixes and improvements (#1242)";
-	"Remove dropper from loader (#1238, #1280)";
-	":clone and :loadavatar are now the same command (#1198)";
-	"Add TextChatService support for command and command autocompletion (#1208)";
-	"Add :sing command (#1217)";
-	"Add new BanMenu UI; add functions+data models to the window code (#1295)";
-	":vote command is now in random order (#1179)";
-	"Add :orderedvote command for voting in order (#1179)";
+	"[v241 2023-12-20 23:59 UTC] Portions (@Dimenpsyonal & @ccuser44)";
```
Comment (moo1210) [inline MainModule/Server/Shared/Changelog.lua:6]: I think this makes it better
```suggestion
	"[v241 2023-12-20 23:59 UTC] Portions (@Dimenpsyonal & @ccuser44 & @moo1210)";
	"The Fixed & Bricked Update";
```

File: MainModule/Server/Shared/Changelog.lua [hunk 2]
```diff
@@ -3,18 +3,24 @@ return {
...
+	"Selene, rojo, aftman and git fixes & upgrades";
 	"";
+	"(Git/kaiserandaxl) Command Box now scrolls (#1307)";
```
Comment (moo1210) [inline MainModule/Server/Shared/Changelog.lua:9]: Cool `git log`; I love Adonis changelogs!

**Review by moo1210 — CHANGES_REQUESTED** [review]
Improved some issues

**Comment by ccuser44** [comment]
Done

**Comment by Dimenpsyonal** [comment]
What does the fixed and bricked update mean

File: MainModule/Server/Shared/Changelog.lua [hunk 3]
```diff
@@ -3,18 +3,25 @@ return {
...
+	"(Git/kaiserandaxl) Add new BanMenu UI; add functions+data models to the window code (#1295)";
+	"(Git/karl-police) :vote command is now in random order (#1179)";
+	"(Git/ccuser44) Add :orderedvote command for voting in order (#1179)";
+	"(Git/dylanjkl) Implemented way to monitor Adonis Teleport status via a Character Attribute named \"ADONIS_isTeleporting\" (#1312)";
+	"(Git/kaiserandaxl) Added all properties and classes to \":GuiView\" (#1314)";
+	"(Git/WalkerOfBacon) Fix :sceleratis (#1305)";
+	"(Git/WalkerOfBacon) Fix :removestats (#1309)";
 	"(Git/ccuser44) Add command to debug anti cheat (#1267)";
 	"(Git/ccuser44) Fix remotespy detection (#1218)";
 	"(Git/ccuser44) Disable protections superseded by RejectCharacterDeletions on new games (#1245)";
```
Comment (moo1210) [inline MainModule/Server/Shared/Changelog.lua:25]: Why are changelogs always `git log` that's not a changelog

Comment (ccuser44) [reply MainModule/Server/Shared/Changelog.lua:25]: What's the issue here?

Comment (moo1210) [reply MainModule/Server/Shared/Changelog.lua:25]: At least half of the commit messages make sense this time I think that might be a new record

Comment (ccuser44) [reply MainModule/Server/Shared/Changelog.lua:25]: You are free to suggest changes

**Comment by Dimenpsyonal** [comment]
Yea that's because I quite a few of them to make them more understandable

File: MainModule/Server/Shared/Changelog.lua [hunk 4]
```diff
@@ -3,18 +3,23 @@ return {
...
-	"Coasterteam various fixes and improvements (#1242)";
-	"Remove dropper from loader (#1238, #1280)";
-	":clone and :loadavatar are now the same command (#1198)";
-	"Add TextChatService support for command and command autocompletion (#1208)";
-	"Add :sing command (#1217)";
-	"Add new BanMenu UI; add functions+data models to the window code (#1295)";
-	":vote command is now in random order (#1179)";
-	"Add :orderedvote command for voting in order (#1179)";
+	"[v241 2023-12-20 23:59 UTC] Portions (@Dimenpsyonal & @ccuser44)";
 	"";
```
Comment (P3tray) [inline MainModule/Server/Shared/Changelog.lua:7]: Add my name in here for fun.

**Review by P3tray — CHANGES_REQUESTED** [review]
Not good enough.

**Comment by Dimenpsyonal** [comment]
> Why are changelogs always `git log` that's not a changelog

@moo1210 then what do you want? it's a log of all changes

**Comment by ccuser44** [comment]
@Dimenpsyonal would have to do this, but it has to be done

**Pull request closed without merging**